src/main/java/step3/domain/Ladder.java

```diff
@@ -0,0 +1,51 @@
+package step3.domain;
+
+import step3.utils.LadderUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Ladder {
+
+    private List<Lines> lines;
+
+    private Ladder(List<Lines> lines) {
+        this.lines = lines;
+    }
+
+    public static Ladder of(int countOfPerson, int maxHeight) {
+        return new Ladder(createLines(countOfPerson, maxHeight));
+    }
+
+    private static List<Lines> createLines(int countOfPerson, int maxHeight) {
+        return IntStream.range(0, maxHeight)
+                .mapToObj(j -> Lines.of(createLine(countOfPerson)))
+                .collect(Collectors.toList());
```

Comment:
```suggestion
        return IntStream.range(0, maxHeight)
                .mapToObj(ignore-> Lines.of(createLine(countOfPerson)))
                .collect(Collectors.toList());
```

변수를 통해 의미를 전달할 수 있겠네요 😄 

만약 해당 값을 사용하지 않는다면 

```suggestion
        return Stream.generate(() -> Lines.of(createLine(countOfPerson)))
                .limit(maxHeight)
                .collect(Collectors.toList());
```

와 같이 작성하실 수 있습니다 😄

Reply:
😎